tests/test_main.py

```diff
@@ -1 +1,20 @@
-# write your code here
+from app.main import is_isogram
+
+
+def test_empty_string_and_result_type_bool():
+    assert is_isogram("") is True
+
+
+def test_repeated_insensitive_letters():
+    assert is_isogram("Apex") is True
+    assert is_isogram("dermatoglyphics") is True
```

Comment:
Here you should check that function is insensitive to letters, so it assume 'T' and 't' as same letter and returns False for `is_isogram("Test")`. Remove current assertions.

Reply:
Fixed